modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -49,6 +55,32 @@
 public class AssetEntryInfoItemFieldSetProviderImpl
 	implements AssetEntryInfoItemFieldSetProvider {
 
+	@Override
+	public InfoFieldSet getInfoFieldSet(AssetEntry assetEntry) {
+		InfoFieldSet infoFieldSet = getInfoFieldSet(AssetEntry.class.getName());
+
+		Locale locale = LocaleThreadLocal.getThemeDisplayLocale();
```

Comment:
We should use the thread local as a last resort. In this case, instead of this we should return all the locales for the frontend to choose which one to use.